tests/plugins/inlinetoolbar/manual/focus.md

```diff
@@ -0,0 +1,11 @@
+@bender-ui: collapsed
+@bender-tags: 4.8.0, bug, inlinetoolbar
+@bender-ckeditor-plugins: wysiwygarea,toolbar,undo,basicstyles,notification,floatingspace,inlinetoolbar,sourcearea,list,link,image
```

Comment:
This manual test does not need that many plugins. Just minimal amount + basicstyles and that's it.